docs/developers/patch.md

```diff
@@ -118,7 +118,7 @@ The `replaceTree()` patch is a convenience operator for `removeTokens` and
 
 ```scala mdoc
 doc.tree.collect {
-  case println @ Term.Apply(Name("println"), _) =>
+  case println @ q"println(..$_)" =>
```

Comment:
It does not matter here as `println` can only have one list of arguments, but I didn't find a built-inway to match a `Term.Apply` with an arbitrary number of elements.

```scala
@ val Term.Apply(expr, _) = q"println()" 
expr: Term = Term.Name(value = "println")

@ val Term.Apply(expr, _) = q"println" 
scala.MatchError: println (of class scala.meta.Term$Name$TermNameImpl)
  ammonite.$sess.cmd48$.<clinit>(cmd48.sc:1)


@ val q"$expr(...$exprssnel)" = q"println()" 
expr: Term = Term.Name(value = "println")
exprssnel: List[Term.ArgClause] = List(Term.ArgClause(values = List(), mod = None))

@ val q"$expr(...$exprssnel)" = q"println" 
expr: Term = Term.Name(value = "println")
exprssnel: List[Term.ArgClause] = List()

@ val q"$expr(..$_)" = q"println" 
scala.MatchError: println (of class scala.meta.Term$Name$TermNameImpl)
  ammonite.$sess.cmd51$.<clinit>(cmd51.sc:1)
```

It looks like the `-nel` suffix has no effect. And having a `Term.Name` match the quasiquote syntax for `Apply` sounds weird.

Reply:
> didn't 3-dots work for arbitrary?

It was too arbitrary. I probably misunderstood the usage of [suffixes](https://scalameta.org/docs/trees/quasiquotes.html#suffixes), but I was expecting `q"println(...$exprssnel)"` not to match `q"println"` as the captured list is empty.

Reply:
never saw those suffixes before. probably convention in the examples on that page.